webapp/channels/src/actions/websocket_actions.jsx

```diff
@@ -39,12 +39,12 @@ import {
     getCustomEmojiForReaction,
     getPosts,
     getPostThread,
-    getMentionsAndStatusesForPosts,
     getThreadsForPosts,
     postDeleted,
     receivedNewPost,
     receivedPost,
 } from 'mattermost-redux/actions/posts';
+import {getBatchedUserProfilesStatusesAndGroupsFromPosts} from 'mattermost-redux/actions/profiles_statuses_groups';
```

Comment:
Nit: name is too large :) Let's have something shorter.

Reply:
I'd suggest something like `getExtraDataForPosts` even though that name isn't great when we already have `PostMetadata` as a concept

Reply:
changed to `addUserIdsForStatusAndProfileFetchingPool`